customer_satisfaction/streamlit_app.py

```diff
@@ -0,0 +1,90 @@
+from re import S
+import streamlit as st
+from inference.predict import predict 
+from PIL import Image
+import pandas as pd 
+
+import pickle 
+def main(): 
+    st.title("End to End Customer Satisfaction Pipeline with ZenML")
```

Comment:
Should be the same title as we decide for the zenfile... still not sure about this one at the moment, though.

Reply:
Yeah, this looks good